### ISSUE TITLE
Code not matching

### ISSUE THREAD

**Issue by HighHeelGeek** [walkingriver/at10dance-angular#47]
@walkingriver I'm attempting to follow along with the exercises in the book. However, the code in the book is not matching the code provided here on GitHub and the code I downloaded from GitHub as a .zip file to use in VScode does not match either the book or GitHub. All three have vast differences, including file name variations and files available. Can you please explain or assist me with this? Thank you very much in advance.

**Comment by walkingriver** [comment]
Hi there, First of all, thanks for reading the book and reaching out. I have to admit that the book is a little outdated right now and is on my “to list” to be updated. What’s in the GitHub repo should be the most accurate and functional. I presume you’re looking at this one? https://github.com/walkingriver/at10dance-angular That repo was updated last summer during a live workshop. Is there a specific question I can help you with?  Thanks,Mike Callaghan From: Kristin Bailey ***@***.***>Date: Thursday, March 9, 2023 at 2:10 PMTo: walkingriver/at10dance-angular ***@***.***>Cc: Michael Callaghan ***@***.***>, Mention ***@***.***>Subject: [walkingriver/at10dance-angular] Code not matching (Issue ***@***.*** I'm attempting to follow along with the exercises in the book. However, the code in the book is not matching the code provided here on GitHub and the code I downloaded from GitHub as a .zip file to use in VScode does not match either the book or GitHub. All three have vast differences, including file name variations and files available. Can you please explain or assist me with this? Thank you very much in advance.—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you were mentioned.Message ID: ***@***.***>

**Comment by HighHeelGeek** [comment]
Yes, do have a specific question. Where do I begin? Here's the comparison of the GitHub files in VS Code and the version I get when I go through the browser. 
![VScodeVersion](https://user-images.githubusercontent.com/104701988/224137865-5c9f3bc2-ac61-47af-a9e4-2e49aab55e49.PNG)
![GitHubwebVersion](https://user-images.githubusercontent.com/104701988/224137907-af1f311d-22be-40e2-84e7-814a9f0db659.PNG)

**Comment by walkingriver** [comment]
May I ask which part of the book you’re working on? And which branch you have open in VS Code? I’m trying to get a sense of where you are and what you’re trying to do. I can tell you that I have not had much success with getting Ionic apps to work in StackBlitz. Granted, I haven’t done it in a while, but it wasn’t working last time I tried. Mike From: Kristin Bailey ***@***.***>Date: Thursday, March 9, 2023 at 2:48 PMTo: walkingriver/at10dance-angular ***@***.***>Cc: Michael Callaghan ***@***.***>, Mention ***@***.***>Subject: Re: [walkingriver/at10dance-angular] Code not matching (Issue #47)Yes, do have a specific question. Where do I begin? Here's the comparison of the GitHub files in VS Code and the version I get when I go through the browser.—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you were mentioned.Message ID: ***@***.***>

**Comment by HighHeelGeek** [comment]
I'm at the intro. I have zero experience with StackBlitz, Ionic, and Angular. I'm literally just trying to follow along with the book and get a feel for the application. I was merely following links provided. If there are better ones to be using I'm happy to take suggestions.

**Comment by walkingriver** [comment]
Would you be open to a 30 minute Zoom call? I’d like to see exactly where the disconnect is. I’m guessing it’s something that has changed either in Angular, Ionic, StackBlitz, my repo, or a combination of those.  From: Kristin Bailey ***@***.***>Date: Thursday, March 9, 2023 at 2:57 PMTo: walkingriver/at10dance-angular ***@***.***>Cc: Michael Callaghan ***@***.***>, Mention ***@***.***>Subject: Re: [walkingriver/at10dance-angular] Code not matching (Issue #47)I'm at the intro. I have zero experience with StackBlitz, Ionic, and Angular. I'm literally just trying to follow along with the book and get a feel for the application. I was merely following links provided. If there are better ones to be using I'm happy to take suggestions.—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you were mentioned.Message ID: ***@***.***>

**Comment by walkingriver** [comment]
I just now realized that this communication is all going through GitHub. Send me an email at ***@***.*** to continue, please. From: ***@***.*** ***@***.***>Date: Thursday, March 9, 2023 at 3:28 PMTo: walkingriver/at10dance-angular ***@***.***>, walkingriver/at10dance-angular ***@***.***>Cc: Mention ***@***.***>Subject: Re: [walkingriver/at10dance-angular] Code not matching (Issue #47)Would you be open to a 30 minute Zoom call? I’d like to see exactly where the disconnect is. I’m guessing it’s something that has changed either in Angular, Ionic, StackBlitz, my repo, or a combination of those.  From: Kristin Bailey ***@***.***>Date: Thursday, March 9, 2023 at 2:57 PMTo: walkingriver/at10dance-angular ***@***.***>Cc: Michael Callaghan ***@***.***>, Mention ***@***.***>Subject: Re: [walkingriver/at10dance-angular] Code not matching (Issue #47)I'm at the intro. I have zero experience with StackBlitz, Ionic, and Angular. I'm literally just trying to follow along with the book and get a feel for the application. I was merely following links provided. If there are better ones to be using I'm happy to take suggestions.—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you were mentioned.Message ID: ***@***.***>
